### ISSUE TITLE
Bootstrap 5 style concerns

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1140]
**Description**
Here are a list of style concerns we can across while QA'ing our site. 

Concerns:

- [ ] All page headers should be center, unless there's a button.
- [ ] All page headers should be `<h1>`.
- [ ] Page bodies should be indented from the navigation bar.
- [ ] All Breadcrumbs should be styled the same.
- [ ] All form buttons should be right aligned.
- [ ] Pagination controls should be right aligned.
- [ ] Bullets in footer list should be removed.
- [ ] When a list is empty there is a custom message that is usually displayed.  This text should be centered and italicized. 
- [ ] Search fields should be centered with a margin.  

** Developer Notes **
Template tags should be heavily used to normalize these above changes.  For example, there can be a tag that will return class necessary to properly style an element.  

```
from django import template

register = template.Library()

@register.simple_tag
def breadcrumbs_classes():
    return "bg-light m-3 ... "
```

```
<nav aria-label="breadcrumb">
  <ol class="breadcrumb {% breadcrumbs_classes %}">
    <li class="breadcrumb-item"><a href="#">Home</a></li>
    <li class="breadcrumb-item active" aria-current="page">Library</li>
  </ol>
</nav>
```

**Comment by okaycj** [comment]
@becky-gilbert @mekline 

Here are the style issues we found in our first QA pass.

**Comment by okaycj** [comment]
> First general comment is on colors...

Becky and I are working on a proposal to resolve this issue.  I can give more context if you're curious.

**Comment by becky-gilbert** [comment]
@mekline WRT this change on the family-facing study list page:
> The bottom header (which I think also is what you see if there are no search results) is going to stop making sense as we merge CHS. Change the text from:
Looking for more ways to contribute to research from home? Check out [Children Helping Science ](https://childrenhelpingscience.com/)for even more studies!
to:
Want to learn more about research with babies & children? Check out the [Resources](https://lookit.mit.edu/resources/) page for activities to try at home and developmental labs near you!

the previous text was included in the translation files and the new text is not. I assume it's ok to include the new text as English only, but let me know what to do if not.

**Comment by mekline** [comment]
Yes, this is fine - we are unavoidably making some text changes in planning for the CHS merger.  My plan is that once we are confident this has settled down, to reach out to our translator volunteers to re-fill.  The only thing to watch here is that in many cases the text will be "close", so having the old translation available will help the translators a lot.  Is it the case that we can grab the old translations from DO or PO files (whichever shows the English <-> other language translation correspondence) off of the github history?

**Comment by mekline** [comment]
Clearing the decks for the next shape - can this be closed @becky-gilbert @okaycj ?

**Comment by becky-gilbert** [comment]
@mekline @okaycj I think it's almost done. CJ addressed the last item on the list in this open PR: #1176 I commented that we could add more to that PR, and I have those changes ready, so I was just waiting on hearing back about that before approving it.
